lib/libnvpair/libnvpair.c

```diff
@@ -199,6 +199,10 @@ nvprint_##type_and_variant(nvlist_prtctl_t pctl, void *private, \
 	return (1); \
 }
 
+#if defined(__GNUC__) && !defined(__clang__)
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wformat-overflow"
+#endif
```

Comment:
```
lib/libnvpair/libnvpair.c:204:9: error: unknown option after '#pragma GCC diagnostic' kind [-Werror=pragmas]
 #pragma GCC diagnostic ignored "-Wformat-overflow"
         ^
```

This appears to be a problem for older versions of gcc from CentOS 7 and Debian 8.